components/table/table-wrapper.js

```diff
@@ -317,7 +321,7 @@ export class TableWrapper extends RtlMixin(LitElement) {
 			const offset = this.type === 'default' ? -3 : 1; // default: -5px top + 2px border, light: 0 top + 1px border
 			const ths = Array.from(table.querySelectorAll('tr.d2l-table-header:not(:first-child) th, tr[header]:not(:first-child) th, thead tr:not(:first-child) th'));
 			ths.forEach((th) => {
-				th.style.top = `${topHeader.clientHeight + offset}px`;
+				th.style.top = `${topHeaderHeight + offset}px`;
```

Comment:
This is the main fix here -- awesomely I already stored this in a variable... just needed to actually use it.